bounties_api/static/apischema.yaml

```diff
@@ -228,6 +228,17 @@ paths:
 
 
   /user/{address}/profile/:
+    post:
+      operationId: user_update_profile_by_address
+      description: 'Update user profile by address'
+      parameters: []
```

Comment:
@iamonuwa you need to add the parameter for address in here

Reply:
Right